framework/tlvf/src/include/tlvf/CmduMessageTx.h

```diff
@@ -27,6 +29,7 @@ class CmduMessageTx : public CmduMessage {
     template <class T> std::shared_ptr<T> addClass() { return msg.addClass<T>(); }
     void reset() { msg.reset(false); }
     bool finalize();
+    size_t element_in_message_mtu_limit(size_t size);
```

Comment:
```suggestion
    size_t max_elements_in_message(size_t size);
```

Reply:
maybe `bus_limit_elements_in_meassage`? It is more accurate...

Reply:
It doesn't matter what the underlying layer is... Just what is the maximal number of logical elements you can add to the message.